### PR TITLE
Add  hint for unknown node type

### PR DESCRIPTION
**Pull request by tplevko**
fix for https://github.com/KaotoIO/kaoto-next/issues/803

Screenshot from 2024-02-06 14-55-50

### CODE REVIEW

File: packages/ui/src/components/Visualization/Canvas/CanvasForm.tsx [hunk 1]
```diff
@@ -88,9 +100,25 @@ export const CanvasForm: FunctionComponent<CanvasFormProps> = (props) => {
...
+            <CardFooter>
+              <Hint>
+                <HintTitle>Node is not a known type</HintTitle>
+                <HintBody>
+                  Configuration properties are not available when the node is not a known type. Delete, or replace this
+                  node with valid node.
+                </HintBody>
+              </Hint>
+            </CardFooter>
+          </Card>
```
Comment (lordrip) [inline packages/ui/src/components/Visualization/Canvas/CanvasForm.tsx:121]: I think this component is starting to grow too big to have here :), maybe it's time to move it to a dedicated file.

**Comment by lhein** [comment]
I'd rephrase the warning message a bit:

**Unknow node type**

The configuration for an unknown node cannot be changed in the configuration form. Please switch to the source code and correct the node type. Another option is to replace the step in the graphical editor but in that case you will probably lose the existing configuration.

File: packages/ui/src/components/Visualization/Canvas/CanvasForm.tsx (hunk 1 above)
Comment (tplevko) [reply packages/ui/src/components/Visualization/Canvas/CanvasForm.tsx]: sure - moved to separate file

**Comment by tplevko** [comment]
Updated - moved to separate file, updated the `Hint` to `Alert`, with variant `warning`. Also changed the message.

With updates:
Screenshot from 2024-02-07 10-01-21

**Comment by lhein** [comment]
you did not change the headline, did you?

**Comment by tplevko** [comment]
> you did not change the headline, did you?

Missed that the headline was also changed - Updated.

**Review by lordrip — APPROVED** [review]
This is a good enhancement @tplevko , thanks 💪
